test/parser/parser_test.cpp

```diff
@@ -390,20 +390,5 @@ TEST_F(ParserTests, CopyTest) {
     LOG_TRACE("%d : %s", ++ii, result->GetInfo().c_str());
   }
 }
-
-// Test that the wrong queries can be detected.
-TEST_F(ParserTests, WrongQueryTest) {
-  std::vector<std::string> queries;
-  queries.push_back("SELECT;");
-  queries.push_back("SELECT *;");
-  // Query with wrong syntax
-  queries.push_back("SECLECT *;");
-
-  // Parsing
-  for (auto query : queries) {
-    EXPECT_THROW(parser::PostgresParser::ParseSQLString(query.c_str()),
-                 Exception);
-  }
-}
```

Comment:
Can you move this to binder_test instead of deleting it?

Reply:
Sure.